### PR TITLE
Change new registration email wording.

### PR DESCRIPTION
**Pull request by jonatanklosko**
Updated as per [the discussion](https://github.com/thewca/worldcubeassociation.org/issues/2413#issuecomment-356879977) over #2413.

@Mollerz how does it look to you?

image

### CODE REVIEW

File: WcaOnRails/config/locales/en.yml [hunk 1]
```diff
@@ -708,10 +708,15 @@ en:
         mail_subject: "You have registered for %{comp_name}"
         #context: "comp_name" contains the competition's name
         registered_html: "You just registered for %{comp_name}."
-        #context: "n_total" will be replaced by the total number of registrations on waiting list, and "people" by the pluralization of "person" in your language (key registrations.registration_info_people.person)
-        on_waiting_list: "Your registration is on the waiting list, which currently has %{n_total} %{people} on it."
+        awaits_approval: "Your registration has been accepted by the system and is pending approval from the organisation team."
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: nit: As part of incorporating in the US, we decided to use US English throughout the website, so this should actually be "organization team", sorry @Mollerz =(

**Comment by jfly** [comment]
Thanks for taking this on so quickly, @jonatanklosko! The code LGTM, but let's wait to hear from @Mollerz before merging it up.

**Comment by Mollerz** [comment]
LGTM!

**Comment by pedrosino** [comment]
Ah, too late here :(

I would say that "Your registration has been received" instead of "accepted by the system", but that's a minor thing.

**Comment by jfly** [comment]
@pedrosino, it's never too late to send in a PR =)

**Pull request merged**